backend/routes/restaurants.js

```diff
@@ -0,0 +1,22 @@
+var express = require("express");
+const services = require("../services");
+var router = express.Router();
+
+
+router.get("/getAll", services.getAllRestaurant);
```

Comment:
Are we following REST principles when it comes to naming endpoints?
If so, it should be "/". But functionally, there is no difference.